### DISCUSSION TITLE
Was phobos Hephaestus based at the very start?

### DISCUSSION THREAD

**Discussion by Zane2b2t** [3arthqu4ke/3arthh4ck#285] [Q&A]
asking this bcs i found u in the mcmod.info of it and i know that phobos used to be a kami skid

**Comment by 3arthqu4ke** [comment]
Yes, it all started because I wanted different offsets for AutoTrap. The first Hephaestus one made a huge trap.

**Reply by Gentleman2292** [reply]
insane

**Comment by Gentleman2292** [comment]
now the real question, does the old base use some code from exeter?

**Reply by Zane2b2t** [reply]
> now the real question, does the old base use some code from exeter?

What base do u mean
There was 3 bases i think
The first one not many know about but now 3arth confirmed the first base was hepatsues meaning the one with exeter code is the 2nd base!!1!1!11!
